jerry-core/ecma/base/ecma-helpers-values-collection.cpp

```diff
@@ -38,7 +38,7 @@ ecma_new_values_collection (const ecma_value_t values_buffer[], /**< ecma-values
 {
   JERRY_ASSERT (values_buffer != NULL || values_number == 0);
 
-  const size_t values_in_chunk = sizeof (ecma_collection_chunk_t::data) / sizeof (ecma_value_t);
+  const size_t values_in_chunk = sizeof (((ecma_collection_chunk_t *)0)->data) / sizeof (ecma_value_t);
```

Comment:
@jiangzidong It is not necessary. You spent time to do this like me, but you were faster to upload. Your solution is also good (the null pointer surprised me, I have never seen similar before). In my comment I just want to symbolize that either work is unnecessary when we do this together (at the same time) without any conversation.

The time you recommended is good for me.